Formula/aws-rotate-iam-keys.rb

```diff
@@ -1,7 +1,7 @@
 class AwsRotateIamKeys < Formula
   desc "Automatically rotate your IAM keys daily"
   homepage "https://aws-rotate-iam-keys.com"
-  url "https://github.com/rhyeal/aws-rotate-iam-keys/archive/v0.9.2.tar.gz"
+  url "https://github.com/rhyeal/aws-rotate-iam-keys/archive/v0.9.5.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/rhyeal/aws-rotate-iam-keys/archive/1176148.tar.gz"
```

Reply:
@veriservus "1176148" represent what exactly ?

Reply:
Well since there is no github tag for 0.9.5 we have to reference it directly with this ID. It basically represents the new version I think. Or old. The important thing is it represents a working version (I tested this locally).